content/release-notes/2023-03-06.md

```diff
@@ -0,0 +1,15 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- UI: Updated the the labels, links, and units shown on the **Project limits** widget on the Neon **Dashboard**.
+
+### Bug Fixes
+
+- API: Adjusted the tags in the Neon API specification that assign endpoints to categories. The new [Neon API reference](https://neon.tech/docs/reference/api-reference) displays one endpoint per page. Multiple tags assigned to an endpoint resulted in endpoints not being listed under their primary category.
+- UI: Fixed the vertical alignment of buttons on authorization dialogs.
+- UI: Adjusted the logic supporting the **Integrations** page in the Neon Console, where you can view and revoke integrations, to only mark Vercel projects as revoked rather than deleting them.
```

Comment:
that's internal details, let's not include it in notes

Reply:
Thank you. Removed.

Reply:
I am sorry for not being clear, I was talking about revoking the integrations

Reply:
@duskpoet So I should keep this part: "Adjusted the logic supporting the **Integrations** page in the Neon Console, where you can view and revoke integrations" and remove "to only mark Vercel projects as revoked rather than deleting them."

Reply:
I think it's better to omit the whole paragraph about the Integrations if the only thing that was changed is revoking logic (which is an internal change)

Reply:
ok!